### PR TITLE
Remove duplicate shutdown log in NEG controller.

### PR DESCRIPTION
**Pull request by sawsa307**
/assign @swetharepakula

### CODE REVIEW

File: pkg/neg/controller.go [hunk 1]
```diff
@@ -371,7 +371,6 @@ func (c *Controller) IsHealthy() error {
 }
 
 func (c *Controller) stop() {
-	c.logger.V(2).Info("Shutting down network endpoint group controller")
```
Comment (sawsa307) [inline pkg/neg/controller.go:374]: Line 341 already has this.

Comment (sawsa307) [reply pkg/neg/controller.go:374]: Added. PTAL!

**Review by swetharepakula — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**